hedera-node/hedera-networkadmin-service-impl/src/test/java/module-info.java

```diff
@@ -12,11 +12,15 @@
     requires com.hedera.node.app.spi.fixtures;
```

Comment:
I thought we are removing the module-info.java from testing

Reply:
I'm not sure what you mean here. Please clarify.

Reply:
you don't need the all file. you can just delete it. Thats what I did and we need to do it to all services the test doesn't need that file at all.

Reply:
The _unit tests_ should not be modules and should not be part of the module path.  There are some compile issues that can come up removing the module-info, however. File Service removed the module-info from the test folder, but also required some other changes (not sure what those were) to make the build still work.

Reply:
I just removed one test from the spi and that's all